### PR TITLE
Add example image to Java agent docs.

### PR DESCRIPTION
**Pull request by bretthoerner**

### CODE REVIEW

File: src/collections/_documentation/clients/java/agent.md [hunk 1]
```diff
@@ -18,3 +18,5 @@ java -agentpath:/path/to/libsentry_agent_linux-x86_64.so -jar app.jar
 You will still need to install and configure the [Sentry Java SDK]({%- link _documentation/clients/java/index.md -%}). In addition, **you must set the** `stacktrace.app.packages` option. Only exceptions that contain at least one frame from your application will be processed by the agent. You can find details about this option [on the configuration page]({%- link _documentation/clients/java/config.md -%}#in-application-stack-frames).
 
 With the SDK configured the agent should automatically enhance your events where applicable.
+
+![Example of local variable state in the Sentry UI](/src/collections/_documentation/clients/java/agent.png)
```
Comment (mitsuhiko) [inline src/collections/_documentation/clients/java/agent.md]: I don't think that's the right way to include the file but I also don't know what the right one is. @cameronmcefee how do we do this?

**Comment by cameronmcefee** [comment]
See e878ca3fb1326de6d32eea0e96e564bd94cb5923. There are a couple ways to do it but this preserves the markdown format

**Comment by bretthoerner** [comment]
Ping @mitsuhiko

**Comment by bretthoerner** [comment]
Thanks @untitaker

**Pull request merged**